plugins/modules/identity/keycloak/keycloak_realm.py

```diff
@@ -0,0 +1,1142 @@
+        type: bool
+        aliases:
+            - adminEventsDetailsEnabled
+
+    admin_events_enabled:
+        description:
+            - Save admin events or not.
+        type: bool
+        aliases:
+            - adminEventsEnabled
+
+    admin_theme:
+        type: str
+        description:
+            - Theme for admin console. Keycloak ships with C(base) and C(keycloak).
+        aliases:
+            - adminTheme
+
+    attributes:
+        type: dict
+        description:
+            - A freeform dict used by Keycloak and extensions to save further attributes for a realm.
+              A non-exhaustive list of options is given below. Some of these are duplications of
+              other API keys -- as noted via references below.
+        suboptions:
+            _browser_header.contentSecurityPolicy:
+                type: str
+                description:
+                    - See I(browser_security_headers).
+            _browser_header.strictTransportSecurity:
+                type: str
+                description:
+                    - See I(browser_security_headers).
+            _browser_header.xContentTypeOptions:
+                type: str
+                description:
+                    - See I(browser_security_headers).
+            _browser_header.xFrameOptions:
+                type: str
+                description:
+                    - See I(browser_security_headers).
+            _browser_header.xRobotsTag:
+                type: str
+                description:
+                    - See I(browser_security_headers).
+            _browser_header.xXSSProtection:
+                type: str
+                description:
+                    - See I(browser_security_headers).
+            actionTokenGeneratedByAdminLifespan:
+                description:
+                    - See I(action_token_generated_by_admin_lifespan)
+            actionTokenGeneratedByUserLifespan:
+                description:
+                    - See I(action_token_generated_by_user_lifespan)
+            actionTokenGeneratedByUserLifespan.idp-verify-account-via-email:
+                description:
+                    - Override user-initiated action lifespan for IdP account email verification, in seconds.
+            actionTokenGeneratedByUserLifespan.reset-credentials:
+                description:
+                    - Override user-initiated action lifespan for resetting credentials/forget password, in seconds.
+            actionTokenGeneratedByUserLifespan.verify-email:
+                description:
+                    - Override user-initiated action lifespan for email verification, in seconds.
+            actionTokenGeneratedByUserLifespan.execute-actions:
+                description:
+                    - Override user-initiated action lifespan for execute actions, in seconds.
+            bruteForceProtected:
+                description:
+                    - See I(brute_force_protected)
```

Comment:
```suggestion
                    - See I(brute_force_protected).
```
the same for each suboption below